packages/app/src/pages/transaction/index.tsx

```diff
@@ -21,10 +23,26 @@ import { TabbedCard, Tab } from '@components/tabbed-card';
 import { finalizeTxSignature } from '@common/utils';
 import { encodeContractCallArgument, getRPCClient, stacksValue } from '@common/stacks-utils';
 
+const broadcastTx = async (serializedTx: Buffer) => {
+  const rpcClient = getRPCClient();
+  return rpcClient.broadcastTX(serializedTx);
+};
+
 interface TabContentProps {
   json: any;
 }
 
+const getInputJSON = (pendingTransaction: TransactionPayload | undefined, identity: any) => {
```

Comment:
```suggestion
const getInputJSON = (identity: any, pendingTransaction?: TransactionPayload) => {
```

Maybe better to switch these params around if one can be considered optional?